app/models/notification_recipient.rb

```diff
@@ -17,6 +17,7 @@ def payload
       :subject    => notification_blueprint.subject,
       :created_at => notification.created_at,
       :group      => notification_blueprint.group,
+      :actions    => notification_blueprint.actions
```

Comment:
how does the actions translate to actual urls given the subject is part of the notification vs notification_blueprint object?